cluster-scope/base/external-secrets.io/externalsecrets/open-cluster-management-observability-alertmanager-config/externalsecret.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: open-cluster-management-observability-alertmanager-config
+  namespace: open-cluster-management-observability
+spec:
+  refreshInterval: 15s
+  secretStoreRef:
+    name: nerc-cluster-secrets
+    kind: ClusterSecretStore
+  target:
+    name: alertmanager-config
+    template:
+      type: Opaque
+      engineVersion: v2
+      data:
+        alertmanager.yaml: |
+          global:
+          route:
+            receiver: default
+            group_by: [cluster, alertname]
+            routes:
+              - receiver: slack-notifications
+                group_by: [alertname, datacenter, app]
+                group_wait: 1d
+                group_interval: 1d
+                repeat_interval: 1d
+                matchers:
+                  - alertname =~ "CustomStoragePersistentVolumeFillingUp|CustomStoragePersistentVolumeFillingUpPredicted|CustomContainerMemoryUsage|CustomContainerCpuUsage"
```

Comment:
I just updated the commit to fix the matchers so that it can match all the alerts I setup in the ConfigMap.